core/task_medium_test.go

```diff
@@ -276,15 +274,32 @@ func TestCreateTaskFromContent(t *testing.T) {
 		So(file, ShouldNotBeNil)
 		So(err, ShouldBeNil)
 		autoStart := true
-		task, err := CreateTaskFromContent(file, &autoStart, nil)
+		task, serr := CreateTaskFromContent(file, &autoStart, nil)
 		So(task, ShouldBeNil)
-		So(err, ShouldNotBeNil)
-		So(err.Error(), ShouldEqual, "invalid character '-' in numeric literal")
+		So(serr, ShouldNotBeNil)
+		So(serr.Errors()[0].Error(), ShouldEqual, "invalid character '-' in numeric literal")
 
 		err = deleteTaskFile(YAML_FILE)
 		So(err, ShouldBeNil)
 	})
 
+	Convey("Bad call to CreateTaskFromContent - no routine", t, func() {
+		err := createTaskFile(JSON_FILE, JSON_FILE_CONTENT)
+		So(err, ShouldBeNil)
```

Comment:
@Alrim42, could you change those tests case to delete task file after it has been successfully created (as a deferred execution)? In this test case, please just put in line 289 `defer deleteTaskFile(JSON_FILE)` and remove the code from lines 299-300, similar for others tests.